string rectImageTopic
geometry_msgs/Transform origin
---
sensor_msgs/Image imageRoi

